### PR TITLE
Update sitemaps

### PR DESCRIPTION
**Pull request by mtruj013**
## Done

- Fixed broken sitemap
- Added missing sitemap links

## QA

- View the site locally in your web browser at: https://canonical-com-1006.demos.haus/sitemap.xml
- Visit each and see that everything works as expected 

## Issue / Card

Fixes https://warthogs.atlassian.net/browse/WD-5612

### CODE REVIEW

File: templates/sitemap-index.xml [hunk 1]
```diff
@@ -9,4 +9,7 @@
   <sitemap>
     <loc>https://canonical.com/partners/sitemap.xml</loc>
   </sitemap>
+  <sitemap>
+    <loc>https://canonical.com/blog/sitemap.xml</loc>
```
Comment (anthonydillon) [inline templates/sitemap-index.xml:13]: I think you need to add a route for this one

**Review by anthonydillon — DISMISSED** [review]
Much better, thanks @mtruj013. Single content in the code.

**Comment by mtruj013** [comment]
Thanks @anthonydillon, fixed! Could you take another look?

**Comment by carkod** [comment]
https://canonical.com/blog/sitemap.xml is still not found?

**Comment by mtruj013** [comment]
@carkod this isn't live yet so that's indeed the expected behavior when you check prod, you need to use the demo URL https://canonical-com-1006.demos.haus/blog/sitemap.xml

**Comment by carkod** [comment]
> @carkod this isn't live yet so that's indeed the expected behavior when you check prod, you need to use the demo URL https://canonical-com-1006.demos.haus/blog/sitemap.xml

Sorry, yeah I should have known.

**Review by carkod — APPROVED** [review]

**Pull request merged**